db/migrate/20151231160846_create_currencies.rb

```diff
@@ -0,0 +1,13 @@
+class CreateCurrencies < ActiveRecord::Migration
+  def change
+    create_table :currencies do |t|
+      t.string :code
+      t.decimal :current_exchange_rate
+      t.datetime :exchange_rate_date
```

Comment:
There are gems, but I'm not sure whether the gems actually absolve us from needing to keep track of this information (though this might be something that we need to denormalize in certain places, like transactions, so that we can account for exchange rates at specific points in time besides "the last time we updated the exchange rate"). 

I was just using the fields that were laid out in the schema document, but I can squash them for now.